_posts/blog/2017-04-28-doing-things-differently.md

```diff
@@ -0,0 +1,42 @@
+## Doing Things Differently
```

Comment:
Suggested front matter for this post... not sure about the tags, but just trying to come up with things we might talk about in other culture posts.
```yaml
---
title: Doing Things Differently
author: Mike Hay
category: Culture
tags: methodology team
---
```